addons/web/static/tests/core/dropdown_tests.js

```diff
@@ -574,22 +575,36 @@ QUnit.module("Components", ({ beforeEach }) => {
         // Click on one
         await click(target, ".one button");
         assert.containsOnce(target, ".dropdown-menu");
+        assert.containsOnce(target, ".one .dropdown-menu");
         // Click on two
         await click(target, ".two button");
-        assert.containsN(target, ".dropdown-menu", 2);
-        // Click on one again
-        await click(target, ".one button");
         assert.containsOnce(target, ".dropdown-menu");
-        assert.containsNone(target.querySelector(".one"), ".dropdown-menu");
+        assert.containsOnce(target, ".two .dropdown-menu");
```

Comment:
Sorry, I did not see this at the last review, but it seems this test lacks the main use case: **"click on (autoOpen=false), hover on (autoOpen=false)"** (it already has "click on (autoOpen=false), hover on (autoOpen=true)").

Summary of all the use cases we want to test:
A. click on (autoOpen=false), hover on (autoOpen=false)
B. click on (autoOpen=false), hover on (autoOpen=true)
C. click on (autoOpen=true), hover on (autoOpen=false)
D. click on (autoOpen=true), hover on (autoOpen=true)

I would say the whole test should have something like this scenario:
- click on one
- hover on two (A)
- hover on three (B)
- click outside
- click on three
- hover on two (C)
- hover on four (D)

Here only B, C & D are covered.